### PR TITLE
OCPBUGSM-35025: reenable unidling ci tests

### PR DESCRIPTION
**Pull request by msherif1234**
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

### CODE REVIEW

File: test/extended/idling/idling.go [hunk 1]
```diff
@@ -286,7 +287,8 @@ var _ = g.Describe("[sig-network-edge][Feature:Idling]", func() {
 			o.Expect(err).ToNot(o.HaveOccurred())
 
 			execPod := e2epod.CreateExecPodOrFail(framework.ClientSet, framework.Namespace.Name, "execpod", nil)
-			o.Consistently(func() error { return tryEchoHTTP(svc, execPod) }, 10*time.Second, 500*time.Millisecond).ShouldNot(o.HaveOccurred())
```
[the remark this answers is not in the document]
Comment (msherif1234) [reply test/extended/idling/idling.go:289]: ok will keep it as it was I am not sure it was adding much value as if it passed the 1st time it should continue to pass, prob was the 1st round

**Comment by bparees** [comment]
@dgoodwin what does TRT want to see before this test gets re-enabled?

**Comment by bparees** [comment]
The fact that this is titled "to debug CI failures" and not "re-enable it because we've fixed it" makes me think this should not merge right now.  I'd expect you to either debug the results using CI jobs run against this PR (not merging it until you're satisfied you've fix it), or update the test to always pass but dump out the debugging info you need, if you need more CI runs than you can reasonably get from a single PR, in order to recreate the issue.  (The latter should only be done w/ careful tracking of a bug to return the test to properly fail, as well as clear comments in place explaining the situation)

**Comment by msherif1234** [comment]
> The fact that this is titled "to debug CI failures" and not "re-enable it because we've fixed it" makes me think this should not merge right now. I'd expect you to either debug the results using CI jobs run against this PR (not merging it until you're satisfied you've fix it), or update the test to always pass but dump out the debugging info you need, if you need more CI runs than you can reasonably get from a single PR, in order to recreate the issue. (The latter should only be done w/ careful tracking of a bug to return the test to properly fail, as well as clear comments in place explaining the situation)


sorry forget to update the title, I was able to repro the issues manually and was able to verify the fix, we hit the issue with ovn ci runs.
the following is list of all idling/unidling test cases that I ran manually to repro and confirm the fix
```
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Unidling should handle many TCP connections by possibly dropping those over a certain bound [Serial] [Suite:openshift/conformance/serial]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Unidling should handle many UDP senders (by continuing to drop all packets on the floor) [Serial] [Suite:openshift/conformance/serial]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Unidling should work with TCP (when fully idled) [Suite:openshift/conformance/parallel]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Unidling should work with TCP (while idling) [Disabled:Broken]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Unidling should work with UDP [Suite:openshift/conformance/parallel]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Idling with a single service and DeploymentConfig should idle the service and DeploymentConfig properly [apigroup:apps.openshift.io] [Disabled:Broken]"
./openshift-tests run-test "[sig-network-edge][Feature:Idling] Idling with a single service and ReplicationController should idle the service and ReplicationController properly [Suite:openshift/conformance/parallel]"
```

**Comment by bparees** [comment]
> sorry forget to update the title, I was able to repro the issues manually and was able to verify the fix, we hit the issue with ovn ci runs.

got it, thanks.  It looks good to me, but i'm still going to ask @dgoodwin to weigh in in case TRT has any particular concerns or requests before we re-enable.  They can remove the hold when they're satisfied.

/approve
/hold

**Comment by bparees** [comment]
whoops, race condition w/ Devan.

/hold cancel

File: test/extended/idling/idling.go [hunk 2]
```diff
@@ -287,6 +288,7 @@ var _ = g.Describe("[sig-network-edge][Feature:Idling]", func() {
 
 			execPod := e2epod.CreateExecPodOrFail(framework.ClientSet, framework.Namespace.Name, "execpod", nil)
 			o.Consistently(func() error { return tryEchoHTTP(svc, execPod) }, 10*time.Second, 500*time.Millisecond).ShouldNot(o.HaveOccurred())
+			o.Expect(err).ToNot(o.HaveOccurred())
```
Comment (jcaamano) [inline test/extended/idling/idling.go:291]: We don't need this check?

**Pull request merged**